### ISSUE TITLE
wordCount and readingTime doesn't work when defaultContentLanguage is set to pl

### ISSUE THREAD

**Issue by lszczygielek** [gohugoio/hugo#7590]
### What version of Hugo are you using (`hugo version`)?

<pre>
$ hugo version
Hugo Static Site Generator v0.74.3-DA0437B4 windows/amd64 BuildDate: 2020-07-23T16:23:30Z
</pre>

### Does this issue reproduce with the latest release?
Yes

I'm creating site in Polish. In my `config.toml` I have:
<pre>
defaultContentLanguage = "pl"
languageCode = "pl"

#rest setup
</pre>

For now I resign from multi-language support and I discovered that reading time and word counting doesn't work in single.html. I've done some testing and I found:

1. When I comment `defaultContentLanguage`, then I see calculation of reading time and word counting, but I need to use en.toml in i18n with polish to translate template - without `defaultContentLanguage` `en` is chosen.
2. When I add post in the English for one post then post in English has calculated values, but post in the Polish hasn't.

Repository with issue: https://github.com/lszczygielek/Hugo-WordCount-ReadingTime-Issue

**Comment by lszczygielek** [comment]
@Lednerb, yes I have i18n/pl.toml.

**Comment by lszczygielek** [comment]
Issue still present in version:
```
$ hugo version
Hugo Static Site Generator v0.75.1-A4A7BAB7 windows/amd64 BuildDate: 2020-09-15T06:48:58Z
```

**Comment by jmooring** [comment]
TLDR
- I am unable to reproduce the problem with v0.82.1.
- The warnings are correct.

```
git clone --recurse-submodules https://github.com/lszczygielek/Hugo-WordCount-ReadingTime-Issue
cd Hugo-WordCount-ReadingTime-Issue
hugo server
```

This generates two warnings:

```
WARN ... "wordCount" has no plural form "many"
WARN ... "readingTime" has no plural form "many"
```

But the values are correct (http://localhost:1313/post/test).

Now add one word of content, because content/post/test.md has nothing beneath the front matter:

```
echo "kot" >> content/post/test.md 
```

This generates no warnings, and the values are correct (http://localhost:1313/post/test).

Now add one _more_ word of content

```
echo "pies" >> content/post/test.md 
```

This generates one warning:

```
WARN ... "wordCount" has no plural form "few"
````

But the values are correct (http://localhost:1313/post/test).

The warnings occur because the [CLDR plural rules](https://unicode-org.github.io/cldr-staging/charts/37/supplemental/language_plural_rules.html) for Polish specify the following cardinal categories: one, few, many, and other. The `few` and `many` categories are missing from the `i18n/pl.toml` file.
